test/data.jl

```diff
@@ -71,6 +71,28 @@ module TestData
     sdf6f = view(df6, UInt64[1, 2])
     @test size(sdf6f) == (2,3)
 
+    #test_group("getindex")
+    v6a = sdf6a[1]
+    @test isa(v6a, DataArray)
+    @test v6a == df6[1, :][1]
+    v6b = sdf6a[:A]
+    @test isa(v6b, DataArray)
+    @test v6b == df6[1, :][:A]
+    sdf6g = sdf6a[[1, 2]]
+    @test isa(sdf6g, SubDataFrame)
+    @test sdf6g[:] == df6[1, [1, 2]]
+    sdf6h = sdf6a[[:A, :B]]
+    @test isa(sdf6h, SubDataFrame)
+    @test sdf6h[:] == df6[1, [:A, :B]]
+    sdf6i = sdf6a[[:A, :B]]
+    @test sdf6h !== sdf6i
+    @test sdf6h.parent[:A] === sdf6i.parent[:A]
+    @test sdf6h.parent[:B] === sdf6i.parent[:B]
+    @test sdf6h.rows === sdf6i.rows
+    sdf6j = sdf6a[:, [:A, :B]]
```

Comment:
Can you test that a copy is returned? Should also test when rows are not `Colon`.